### PR TITLE
Feature/newsletter change

### PR DESCRIPTION
**Pull request by smiechuskce**
Updated newsletter endpoint URL with minor changes is fields.

## Motivation and context
#193 

## How has this been tested?
Run app, open home page and try to register to the newsletter.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -196,38 +196,49 @@ else
...
                     </div>
-                </div>
+                    <div class="row">
+                        <div class="col-md-12 col-sm-12">
+                            <div class="mc-field-group input-group checkbox">
+                                <label>
+                                    <input type="checkbox" name="gdpr" id="mce-GDPR" class="required">
+                                    <strong>Zgoda na przetwarzanie danych</strong>: Zostawiając tutaj swojego maila zgadzasz się na postanowienia regulaminu dostępnego tutaj:
+                                    <a href="https://2018.devadventcalendar.pl/regulamin.pdf">Regulamin.pdf</a>
+                                </label>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: please do not use https://2018.devadventcalendar.pl/regulamin.pdf - use https://devadventcalendar.pl/regulamin.pdf (add if it does not exist)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css [hunk 2]
```diff
@@ -1379,6 +1379,11 @@ h1,h2,h3,h4,h5,h6{
 	transform: translate(-50%, -50%);
 	top: 50%;
 }
+
+.checkbox_field input {
+    margin: 10px 10px 0 0;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css]: @darek-grund could you check this style? #195

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -196,38 +196,49 @@ else
...
+                                    <a href="https://devadventcalendar.pl/regulamin.pdf">Regulamin.pdf</a>
+                                </label>
+                            </div>
+                        </div>
+                    </div>
+                    <div style="display:none;">
+                        <label for="hp">HP</label><br/>
+                        <input type="text" name="hp" id="hp"/>
+                    </div>
+                    <input type="hidden" name="list" value="qGHg763QCyy6QqNG50NV763i0Q"/>
```
Comment (patrikulus) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: What is this magic value?

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css (hunk 2 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css]: Forgot to remove it. I've already done it.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml (hunk 3 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: I ask Michał Gellert, but in my opinion, MailChimp uses it. The link was changed yesterday.

Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml]: Yes, this is an automatically generated value by Sendy (MailChimp).

**Comment by WTobor** [comment]
Could we change the validation message to English?
image

**Comment by WTobor** [comment]
Why there is an error message in different places?

**Comment by WTobor** [comment]
There is no error message but there is no success :(
image

**Comment by WTobor** [comment]
image

**Pull request merged**